content/en/docs/tasks/configure-pod-container/configure-projected-volume-storage.md

```diff
@@ -8,7 +8,7 @@ weight: 70
 ---
 
 {{% capture overview %}}
-This page shows how to use a [`projected`](/docs/concepts/storage/volumes/#projected) volume to mount several existing volume sources into the same directory. Currently, `secret`, `configMap`, and `downwardAPI` volumes can be projected.
+This page shows how to use a [`projected`](/docs/concepts/storage/volumes/#projected) volume to mount several existing volume sources into the same directory. Currently, `secret`, `configMap`, `downwardAPI` and `serviceAccountToken` volumes can be projected.
```

Comment:
oh, forgot to mention, `serviceAccountToken` can be projected but ... unlike the other options, it is not a volume type by itself.